build-logic/src/main/java/com.streamplayer.android-library.gradle.kts

```diff
@@ -17,6 +17,7 @@ plugins {
     id("com.streamplayer.dokka")
     id("org.jetbrains.kotlinx.kover")
     id("com.streamplayer.detekt")
+    id("org.jetbrains.kotlin.plugin.serialization")
```

Comment:
**#question**
Why all library modules should depends on `serialization` plugin?